### ISSUE TITLE
Set up for coc-syntax?

### ISSUE THREAD

**Issue by 7erra** [neoclide/coc-sources#72]
The documentation needs more information on how to set up coc to use the syntax file for autocompletion. The suggested help file does not contain specific enough information to be useful.

**Comment by La1ty** [comment]
I think this issue could have some help: #73

To get `coc-syntax` work:
1. ensure `syntax enable` or `syntax on`, and check the output of `:echo syntaxcomplete#OmniSyntaxList()`

> some plugins may disable syntax, like `nvim-treesitter`  
``` lua
require'nvim-treesitter.configs'.setup {
    highlight = {
        enable = true,
        additional_vim_regex_highlighting = false,  -- this option will disable syntax of Vim
    },
}
```

2. install a `coc.nvim` $\le$ v0.0.81

> You may need to replace `coc#pum#visible()` with `pumvisible()` in many keymaps.

**Comment by chemzqm** [comment]
Fixed by #74
